privaterelay/views.py

```diff
@@ -209,8 +216,8 @@ def _handle_fxa_profile_change(authentic_jwt, social_account, event_key):
 
 
 def _handle_fxa_delete(authentic_jwt, social_account, event_key):
-    # TODO?: loop over the user's relay addresses and create hard bounce
-    # receipt rules for them? Note: can't just do this in RelayAddress.delete
+    # TODO: Loop over the user's relay addresses and manually call delete()
+    # to create hard bounce receipt rules in SES,
     # because cascade deletes like this don't necessarily call delete()
     deleted_objects = social_account.user.delete()
```

Comment:
Maybe we should call `_delete_invitation` here as well, or add `_delete_invitation` on the non-exception case for iterating through the FX events [here](https://github.com/mozilla/fx-private-relay/pull/347/files#diff-f6ad49a624db6b808c53bc30d2ca6713R159-R161).

Reply:
Added.